src/pages/login/LoginForm.tsx

```diff
@@ -36,48 +47,47 @@ class LoginForm extends React.Component<LoginProps, State> {
   }
 
   render() {
+    const {
+      EMAIL,
+      PASSWORD,
+      SUBMIT_ERROR
+    } = LoginFormField;
     return (
       <Formik
         initialValues={{
-          loginEmail: '',
-          loginPassword: '',
-          authenticationError: '',
+          [EMAIL]: '',
+          [PASSWORD]: '',
+          [SUBMIT_ERROR]: '',
         }}
         validationSchema={LoginSchema}
         onSubmit={this.handleSubmit}
       >
         {({ errors, touched, setFieldValue, submitForm, isSubmitting }) => (
-          <Form className="mt-5" method="POST">
+          <Form method="POST">
```

Comment:
is the method="POST" necessary?

Reply:
yeah if we don't have method="POST" it assumes get request and will append the input into into the url parameters

Reply:
👍

Reply:
reactstrap's `Form` component is pretty basic so it needs things like this

Reply:
Why not use Formik's Form component?

Reply:
actually reactstrap has a nifty `tag` prop that takes an a component.  was able to alias formik's `Form` as `FormikForm` and added it in as the tag prop

Reply:
also we're using as much reactstrap as possible so we get the right stylings right off the bat